ingest/v2.md

```diff
@@ -0,0 +1,96 @@
+tags: overview
+description: Use the TelemetryDeck Ingest API to send signals to TelemetryDeck
+lead: Use the TelemetryDeck Ingest API to send signals to TelemetryDeck
+order: 0
+headerImage: /img/ingestv2.jpg
+---
+
+Usually you'll send signals to TelemetryDeck using one of our SDKs. However, if you're working with a language or framework that we don't have an SDK for, you can send signals directly to our Ingest API.
+
+## Ingest Endpoint
+
+Please send your signals as POST request to our ingestion server at `https://nom.telemetrydeck.com/v2/`, with the headers `Content-Type: application/json` and `charset=utf-8`.
+
+Here's an example in cURL:
+
+```bash
+curl -X POST "https://nom.telemetrydeck.com/v2/" \
+    -H "Content-Type: application/json; charset=utf-8" \
+    --data '[{"appID":"AAAA-BBBBBBBB-CCCC-DDDD",
+              "clientUser":"myClientUserHash",
+              "type":"Curl.Development.apiCall"}]' \
+    https://nom.telemetrydeck.com/v2/
+```
+
+## Body Structure
+
+The post body should be an **array of JSON objects**, because you can send multiple signals at once. Each signal object **must** have these properties:
+
+| Property     | Type   | Description                                                                                                                                                                                                                              |
+| ------------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `appID`      | string | Your app's ID                                                                                                                                                                                                                            |
+| `clientUser` | string | A hash of the user's ID. This should always be the same for the same user.                                                                                                                                                               |
+| `type`       | string | The type of signal. While it is not enforced, we recommend structuring your signal names in namespaces separated by dots, with the signal type beginning with a lower case letter and any namespaced beginning with an uppercase letter. |
+
+The following properties are optional:
+
+| Property     | Type   | Description                                                                                 |
+| ------------ | ------ | ------------------------------------------------------------------------------------------- |
+| `sessionID`  | string | The user's session ID. This should be the same value for the same session/user combination. |
+| `payload`    | object | A JSON object with additional data.                                                         |
```

Comment:
Does `payload` support deeply nested objects or is it a flat dictionary of string fields?